Object Store/func.py

```diff
@@ -1,67 +1,85 @@
 import io
-import oci
-import re
 import os
+import gzip
 import json
-import sys
-import requests
 import logging
-import time
-import gzip
 
-from fdk import response
+import oci
+import requests
 
 
-def handler(ctx, data: io.BytesIO=None):
+def handler(ctx, data: io.BytesIO = None) -> None:
     try:
         body = json.loads(data.getvalue())
-        data = body.get("data", {})
-        additional_details = data.get("additionalDetails", {})
-        namespace = additional_details.get("namespace")
-        bucket = additional_details.get("bucketName")
-        obj = data.get("resourceName")
-        eventtime = body.get("eventTime")
+    except (Exception, ValueError) as ex:
+        logging.getLogger().info(str(ex))
+        return
 
-        source = "Oracle Cloud" #adding a source name.
-        service = "OCI Logs" #adding a servicen name.
+    data = body.get("data", {})
+    additional_details = data.get("additionalDetails", {})
 
-        datafile = request_one_object(namespace, bucket, obj)
-        data = str(datafile,'utf-8')
+    namespace = additional_details.get("namespace")
+    if not namespace:
+        logging.getLogger().error("No namespace provided")
+        return
+
+    bucket = additional_details.get("bucketName")
+    if not bucket:
+        logging.getLogger().error("No bucket provided")
+        return
+
+    resource_name = data.get("resourceName")
+    if not resource_name:
+        logging.getLogger().error("No obj provided")
+        return
 
-        datadoghost = os.environ['DATADOG_HOST']
-        datadogtoken = os.environ['DATADOG_TOKEN']
+    event_time = body.get("eventTime")
 
-        for lines in data.splitlines():
-            logging.getLogger().info("lines " + lines)
-            payload = {}
-            payload.update({"host":obj})
-            payload.update({"time": eventtime})
+    source = "Oracle Cloud"  # Adding a source name.
```

Comment:
nit: constants in python are written as all caps snake case 
```suggestion
    SOURCE = "Oracle Cloud"  # Adding a source name.
```